gremlin-go/driver/logger.go

```diff
@@ -55,12 +55,12 @@ type defaultLogger struct {
 }
 
 // Log writes a message to the defaultLogger.
-func (logger *defaultLogger) Log(verbosity LogVerbosity, v ...interface{}) {
+func (logger *defaultLogger) Log(_ LogVerbosity, v ...interface{}) {
 	log.Print(v...)
 }
 
 // Logf writes a formatted message to the defaultLogger.
-func (logger *defaultLogger) Logf(verbosity LogVerbosity, format string, v ...interface{}) {
+func (logger *defaultLogger) Logf(_ LogVerbosity, format string, v ...interface{}) {
 	log.Printf(format, v...)
 }
```

Comment:
@simonz-bq I just noticed we are ignoring verbosity, is this intentional?

Reply:
Yes.

The `logHandler` deals with deciding whether or not to pass in the severity of the message.

The interface is just so anyone that implements their own logger has access to the verbosity if they want to do something with it.